src/libData/AccountData/TransactionReceipt.cpp

```diff
@@ -109,6 +109,16 @@ void TransactionReceipt::AddTransition(const Address& addr,
   m_tranReceiptObj["transitions"].append(_json);
 }
 
+void TransactionReceipt::AddAccepted(bool accepted) {
+  m_tranReceiptObj["accepted"] = accepted;
+}
+
+void TransactionReceipt::AddAcceptedForLastTransition(bool accepted) {
+  LOG_MARKER();
+  m_tranReceiptObj["transitions"][(m_tranReceiptObj["transitions"].size() - 1)]
```

Comment:
Is there a possibility of transitions array being empty?

Reply:
This will only be called when there were `Last Transition`, so it won't be empty. But agree to add a check to be safer.

Reply:
Addressed @ansnunez